egs2/cmu_indic/tts1/conf/train.yaml

```diff
@@ -0,0 +1 @@
+tuning/train_gst_tacotron2.yaml
```

Comment:
Is this recipe single speaker or multi speaker?
If single speaker, we do not need to use xvector, please remove x-vector-based configs.

FYI: Placing minimum configurations is fine.
https://github.com/espnet/espnet/tree/master/egs2/ruslan/tts1/conf/tuning